state/cleanup.go

```diff
@@ -752,10 +785,64 @@ func (st *State) cleanupUnitsForDyingApplication(applicationname string, cleanup
 	return nil
 }
 
+func (st *State) cancelCleanupForceDestroyedUnit(appName string) error {
```

Comment:
Maybe we could call this `cancelCleanupForceDestroyedAppUnits`

Since we are passing in an app name

Reply:
this method has been removed